test/models/user_test.rb

```diff
@@ -0,0 +1,57 @@
+require "test_helper"
+
+describe User do
+  describe 'relations' do
+    it 'can set the user id through new vote' do
+      # Create two models
+      @user = User.create!(username: 'puppy')
```

Comment:
In the context of these tests, within a test (within one "it" block), an instance variable (`@user`) has a larger variable scope than a local variable (such as what `user` could be)... and a local variable would work just fine. In that sense, when there's a question of "should I use an instance variable or a local variable," the preference is to pick the one with smallest scope that will still work (aka pick the local variable)